### PR TITLE
fix bad url in std-file-readText.dd

### PR DESCRIPTION
**Pull request by timotheecour**

### CODE REVIEW

File: changelog/std-file-readText.dd [hunk 1]
```diff
@@ -1,7 +1,7 @@
 readText now checks BOMs
 
 $(REF readText, std, file) now checks for a
-$(HTTP https://en.wikipedia.org/wiki/Byte_order_mark, BOM). If a BOM is present
+$(HTTPS en.wikipedia.org/wiki/Byte_order_mark, BOM). If a BOM is present
```
Comment (wilzbach) [inline changelog/std-file-readText.dd]: Ideally we phase out this entire HTTP/HTTPS non-sense, and just use LINK2, but Andrei has been against this in the past.

**Comment by wilzbach** [comment]
Merged manually due to https://github.com/braddr/d-tester/issues/70

File: changelog/std-file-readText.dd (hunk 1 above)
Comment (timotheecour) [reply changelog/std-file-readText.dd]: breaking down links like that is UGLY and contrary to having readable docs when they're un-rendered (eg we can't simply copy paste these links to/from a browser); and it's error prone as examplified here

Comment (wilzbach) [reply changelog/std-file-readText.dd]: See also:

https://github.com/dlang/phobos/pull/4411#issuecomment-226326952
https://github.com/dlang/phobos/pull/4951#issuecomment-266670581

In short:
- at least there's `WEB` anymore
- The Ddoc issue has been fixed, so in theory we could try this again, but I'm not sure whether it's worth the pain.

Though maybe we can can @andralex's approval this time?

Comment (andralex) [reply changelog/std-file-readText.dd]: Typing `https://` is awkward and integrating it in a macro is sensible. We could and should fix matters on the ddoc side; the issue of line breakage in macros is not singular to this one. Until then please leave HTTP and HTTPS where they are. Thanks.

**Pull request merged**